samples/variadic.go

```diff
@@ -0,0 +1,25 @@
+// Variadic
+package main
+
+import (
+	"fmt"
+)
```

Comment:
Remove parens from single import for consistency with the rest of the samples.

new link: https://goplay.space/#-G75RWA1DW1

Reply:
Done.